errors/errors.go

```diff
@@ -5,8 +5,9 @@ import (
 
 	"github.com/pkg/errors"
 )
-
+// Global error registry, codes 1-99 are reserved for global errors, 0 is reserved for non-errors
```

Comment:
req: 0 is reserved for internal errors where the description string is not exposed to any client

Reply:
@alpe We had to change this as discussed in the initial error proposal PR fix with @ethanfrey, here's the link with a detailed explanation: https://github.com/iov-one/weave/pull/308#discussion_r255423652 . I know it is confusing, but it is what it is. Basically it has to do with 0 being a success abci code.

Reply:
:point_up: